cmd/jaeger/internal/integration/integration.go

```diff
@@ -109,6 +109,28 @@ func createStorageCleanerConfig(t *testing.T, configFile string) string {
 	trace_storage := query["trace_storage"].(string)
 	extensions["storage_cleaner"] = map[string]string{"trace_storage": trace_storage}
 
+	jaegerStorage, ok := extensions["jaeger_storage"].(map[string]interface{})
+	require.True(t, ok)
+
+	switch storage {
+	case "elasticsearch":
+		elasticsearch, ok := jaegerStorage["elasticsearch"].(map[string]interface{})
+		require.True(t, ok)
+		esMain, ok := elasticsearch["es_main"].(map[string]interface{})
+		require.True(t, ok)
+		esMain["service_cache_ttl"] = "1ms"
```

Comment:
I added an additional YAML processing to set "service_cache_ttl" to 1 ms, which automatically deletes the service cache after 1 microsecond. I found this method more convenient. If this method looks good to you, I will remove the purge method from the cache.

```yaml
# Processed YAML
      es_main:
        server_urls: http://localhost:9200
        log_level: "error"
        index_prefix: "jaeger-main"
        use_aliases: false
        create_mappings: true
        service_cache_ttl: 1ms
```

Reply:
In V1, we delete all indices without clearing the cache, still all tests are passing.

Reply:
in V1 we probably reinitialize the write for every test, so caches are dropped automatically. That's actually how we should fix it - change v1 to NOT recreate everything for every test.

ttl=1ms might work, I think we retry all queries for up to 100sec.

Reply:
> change v1 to NOT recreate everything for every test.

should I create a New PR for this?

> ttl=1ms might work, I think we retry all queries for up to 100sec.

Yes